src/main/resources/env.schema.json

```diff
@@ -0,0 +1,195 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "OTP Middleware JSON schema defined to validate the environment configuration.",
+  "description": "Note: Required values must be defined under the 'required' array at the bottom of this schema. Default values are examples only and are not applied at runtime.",
+  "type": "object",
+  "properties": {
+    "AUTH0_API_CLIENT": {
+      "type": "string",
+      "default": "test-auth0-client-id",
+      "description": "API client id required to authenticate with Auth0."
+    },
+    "AUTH0_API_SECRET": {
+      "type": "string",
+      "default": "test-auth0-secret",
+      "description": "API secret id required to authenticate with Auth0."
+    },
+    "AUTH0_DOMAIN": {
+      "type": "string",
+      "default": "test.auth0.com",
+      "description": "Auth0 tenant URL."
+    },
+    "AWS_PROFILE": {
+      "type": "string",
+      "default": "default",
+      "description": "AWS profile for credentials"
+    },
+    "AWS_API_SERVER": {
+      "type": "string",
+      "default": "aws-api-id.execute-api.us-east-1.amazonaws.com",
+      "description": "For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server."
+    },
+    "AWS_API_STAGE": {
+      "type": "string",
+      "default": "stage-name",
+      "description": "For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server."
+    },
+    "BUGSNAG_API_KEY": {
+      "type": "string",
+      "default": "bugsnag-api-key",
+      "description": "A valid Bugsnag authorization token."
+    },
+    "BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_MINUTES": {
+      "type": "integer",
+      "default": "5",
+      "description": "Bugsnag event request frequency."
+    },
+    "BUGSNAG_EVENT_JOB_DELAY_IN_MINUTES": {
+      "type": "integer",
+      "default": "1",
+      "description": "Bugsnag event job frequency."
+    },
+    "BUGSNAG_PROJECT_JOB_DELAY_IN_MINUTES": {
+      "type": "integer",
+      "default": "1",
+      "description": "Bugsnag project job frequency."
+    },
+    "BUGSNAG_ORGANIZATION": {
+      "type": "string",
+      "default": "bugsnag-organization-id",
+      "description": "A valid Bugsnag organization id."
+    },
+    "BUGSNAG_PROJECT_NOTIFIER_API_KEY": {
+      "type": "string",
+      "default": "bugsnag-project-api-key-id",
+      "description": "A valid Bugsnag project API key."
+    },
+    "BUGSNAG_REPORTING_WINDOW_IN_DAYS": {
+      "type": "integer",
+      "default": "14",
+      "description": "Specifies how far in the past events should be retrieved."
+    },
+    "DEFAULT_USAGE_PLAN_ID": {
+      "type": "string",
+      "default": "usage-plan-id",
+      "description": "AWS API gateway default usage plan used when creating API keys for API users."
+    },
+    "OTP_ADMIN_DASHBOARD_FROM_EMAIL": {
```

Comment:
I think you've updated most of the config fields since dev was merged in, but I think we're still missing the optional `MONITORED_COMPONENTS`.

Reply:
yeah, I think that is the last one, updated.